### ISSUE TITLE
Modal windows

### ISSUE THREAD

**Issue by trmendes** [caressofsteel/hugo-story#47]
Is there a way to have a modal window and open youtube videos or text on it?

With a modal window I would like to have:

1) A more complete text displayed when clicking on “Learn More” on a spotlight
2) A gallery with videos that, when the user clicks a modal window, you display the video.

Is it possible to have such behaviors with the current theme?
If not, is it OK to have it as a feature request?

Thanks

**Comment by caressofsteel** [comment]
I don't believe the functionality to achieve your goal is in the code right now, but it shouldn't be too difficult to add. There are plenty of libraries with modal functions you could add to the project. Feel free to give it a shot and submit a PR if you are successful.
